src/k8s/pkg/k8sd/features/localpv/localpv_test.go

```diff
@@ -0,0 +1,155 @@
+package localpv_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	"k8s.io/utils/ptr"
+
+	"github.com/canonical/k8s/pkg/client/helm"
+	helmmock "github.com/canonical/k8s/pkg/client/helm/mock"
+	"github.com/canonical/k8s/pkg/k8sd/features/localpv"
+	"github.com/canonical/k8s/pkg/k8sd/types"
+	snapmock "github.com/canonical/k8s/pkg/snap/mock"
+)
+
+func TestDisabled(t *testing.T) {
+	t.Run("HelmApplyFails", func(t *testing.T) {
+		g := NewWithT(t)
+
+		applyErr := errors.New("failed to apply")
+		helmM := &helmmock.Mock{
+			ApplyErr: applyErr,
```

Comment:
Have you considered using the helm client mock from the snap: https://github.com/canonical/k8s-snap/blob/main/src/k8s/pkg/snap/snap.go#L278?

Reply:
I think this _is_ the helm client mock, right? so we have a snap mock here to which we're passing a helm mock. when `HelmClient()` is called on that snap mock, our mocked helm will be called, right?

Reply:
Yes, you are right!